docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+# Avalanche - Creating a Genesis File For Your Subnet
+
+## Introduction
+
+Hi there! In this tutorial, we will be learning how to create a custom genesis file for your Subnet.
+
+We will start by deploying a very simple Subnet using the Subnet wizard. Then we will learn about the genesis file and how to customize it for our needs.
+
+Next, we will create a simple game!
+
+We will create and deploy a new Subnet using our custom genesis file for the game. Finally, we will be deploying the Game contract to the Subnet.
+
+The Game contract has a simple function called `play` that accepts `LUCK` tokens (native currency of our Subnet) and the contract will either lock the tokens or **mint** 2x the amount of tokens to the player. Yes, it's going to mint native tokens.
+
+At the end of the tutorial, you will *hopefully* have a solid understanding of how to customize the genesis file of your Subnet.
+
+I hope you enjoy this tutorial!
+
+## Table of Contents
+
+- [Avalanche - Creating a Genesis File For Your Subnet](#avalanche---creating-a-genesis-file-for-your-Subnet)
+  - [Introduction](#introduction)
+  - [Table of Contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+    - [Metamask](#metamask)
+    - [Avalanche CLI](#avalanche-cli)
+- [Creating and deploying a Simple Subnet](#creating-and-deploying-a-simple-Subnet)
+  - [What Is a Subnet?](#what-is-a-Subnet)
+    - [Subnets](#Subnets)
+    - [Genesis File](#genesis-file)
+  - [Creating a Simple Subnet](#creating-a-simple-Subnet)
+      - [Choose Your VM](#choose-your-vm)
+      - [Chain Id](#chain-id)
+      - [Token Symbol](#token-symbol)
+      - [Gas Configuration](#gas-configuration)
+      - [Airdropping Native Tokens](#airdropping-native-tokens)
+      - [Adding a Custom Precompile To Modify the EVM](#adding-a-custom-precompile-to-modify-the-evm)
+        - [Native Minting](#native-minting)
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) the nested `<li>`s have too many spaces in the indentation.

> MD007/ul-indent: Unordered list indentation [Expected: 6; Actual: 8]

[More info: MD007](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md007---unordered-list-indentation)